internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -91,6 +91,7 @@ const (
 	configmapAnnotation                 = "unsupported.agent-install.openshift.io/assisted-service-configmap"
 	imageServiceSkipVerifyTLSAnnotation = "unsupported.agent-install.openshift.io/assisted-image-service-skip-verify-tls"
 	allowUnrestrictedImagePulls         = "unsupported.agent-install.openshift.io/assisted-service-allow-unrestricted-image-pulls"
+	disableImageServiceAnnotation       = "unsupported.agent-install.openshift.io/disable-image-service"
```

Comment:
Is this really going to be unsupported?
Are the customers using this going to be happy about using an "unsupported" annotation to make this work?

Reply:
Any resolution on this?